docs/DeveloperGuide.md

```diff
@@ -363,7 +360,7 @@ The following activity diagram summarizes what happens when a user executes a ne
     * Pros: Easy to implement.
     * Cons: The search is limited to matching names. If there are many items containing that keyword, the search may not be efficient.
 
-* **Alternative 2:** Find items in the current list that matches the keyword and an attribute e.g. tag.
+* **Alternative 2:** Find items in the current list that matches the keyword, and an attribute e.g. tag.
     * Pros: More efficiently retrieve the item needed.
```

Comment:
Should this be "Users would be able to retrieve a specific item more efficiently."

Reply:
Alright will update this